### PR TITLE
bayestestR_v2

### PR DESCRIPTION
**Pull request by DominiqueMakowski**
A tiny post just to invite people to check out the package :)

@strengejacke @mattansb @pdwaggoner @IndrajeetPatil

### CODE REVIEW

**Comment by mattansb** [comment]
Looks good (:

File: content/posts/bayestestR_v2.Rmd [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+```{r message=FALSE, warning=FALSE, echo=FALSE, out.width = "200px"}
+include_graphics("https://github.com/easystats/bayestestR/raw/master/man/figures/logo.png")
+```
+
+
+
+## bayestestR v2 is here!
+
+As you might have heard from [previous posts](https://easystats.github.io/blog/posts/), [we](https://github.com/orgs/easystats/people) have recently started to collaborate around the new [**easystats project**](https://github.com/easystats), a set of packages designed to make your life easier. One of the package, [**bayestestR**](https://github.com/easystats/bayestestR), has just been updated on CRAN.
```
Comment (pdwaggoner) [inline content/posts/bayestestR_v2.Rmd]: I would maybe add: "...around the new easystats project, which is a suite of packages..." Just a few word changes/ideas

Comment (pdwaggoner) [reply content/posts/bayestestR_v2.Rmd]: Also, in the next sentence, package should be plural: "One of the packageS..."

File: content/posts/bayestestR_v2.Rmd [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+And this release is so **packed with new features** and **improvements** that it would impossible to present them all in one post! We have added the computation of different types of [**Bayes factors (BF)](https://easystats.github.io/bayestestR/articles/bayes_factors.html), new [**plotting methods**](https://github.com/easystats/see#bayestestr) available in the `see` package, a meta-function, [`describe_posterior`](https://easystats.github.io/bayestestR/reference/describe_posterior.html), that compute **everything at once**, and more...
```
Comment (pdwaggoner) [inline content/posts/bayestestR_v2.Rmd]: "...that computeS..." (add an S to "compute")

**Comment by pdwaggoner** [comment]
Just a couple grammatical changes if its not too late... Mostly looks great though!

**Comment by mattansb** [comment]
@pdwaggoner Took care of those for ya!

I think this is ready to merge / build / publish etc...

**Comment by pdwaggoner** [comment]
Many thanks @mattansb !

**Comment by strengejacke** [comment]
Shall we wait until beginning of next week? This will probably reach more people.

**Comment by mattansb** [comment]
Yes, that's probably best. Do you want to space out the next post as well?

**Comment by strengejacke** [comment]
Yes, can do. But what would be a good timing? Mid of week, or a week later?

**Comment by mattansb** [comment]
If possible, no later than Monday June 10th (when I plan on showcasing `bayestestR` in class and would like to have the emmeans blog post ready to send to students :)

**Comment by DominiqueMakowski** [comment]
In that case, I'd say releasing this one Monday morning and then Mattan's one Wednesday/Thursday is ok?

**Comment by mattansb** [comment]
Monday and Wednesday sounds good :)

**Comment by DominiqueMakowski** [comment]
Merging in 8 minutes

**Pull request merged**